internal/bminventory/inventory_test.go

```diff
@@ -11708,6 +11709,143 @@ var _ = Describe("V2UpdateHostIgnition", func() {
 	})
 })
 
+var _ = Describe("V2UpdateHostIgnition - with rhsso auth", func() {
+	var (
+		authCtx      context.Context
+		bm           *bareMetalInventory
+		db           *gorm.DB
+		ctx          = context.Background()
+		clusterID    strfmt.UUID
+		hostID       strfmt.UUID
+		infraEnvID   strfmt.UUID
+		dbName       string
+		userName1    = "test_user_1"
+		userName2    = "test_user_2"
+		mockOcmAuthz *ocm.MockOCMAuthorization
+		payload      *ocm.AuthPayload
+	)
+
+	BeforeEach(func() {
+		db, dbName = common.PrepareTestDB()
+		clusterID = strfmt.UUID(uuid.New().String())
+		hostID = strfmt.UUID(uuid.New().String())
+		infraEnvID = strfmt.UUID(uuid.New().String())
+
+		_, cert := auth.GetTokenAndCert(false)
```

Comment:
Let's have a test utility to build the necessary auth environment for tests. It is used widely now

Reply:
Good idea, I'll add it in a consecutive PR to use it in all tests.

Reply:
added an helper to auth_handler_test_utils